crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,17 @@ def addons
       end
     end
 
+    def repocheck
+      {
+        cloud: {
+          available: repo_version_available("suse-openstack-cloud", "7") == 0
```

Comment:
oh I saw you posted the same question already, lets change 7 to 8 for master now and the other one we have to adapt when we work on the next upgrade. by then we will likely know the sles version